config/autoload.php

```diff
@@ -0,0 +1,17 @@
+<?php
+spl_autoload_register('myAutoLoader');
+
+function myAutoLoader($class){
+        if(file_exists("models/$class.php"))
+        {
+            include_once("models/$class.php");
+        }
+        else if (file_exists("controllers/$class.php"))
+        {
+            include_once("controllers/$class.php");
+        }
+        else if (file_exists("classes/$class.php"))
+        {
+            include_once("classes/$class.php");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ["include_once" statement detected. File manipulations are discouraged. Statement is not a function, no parentheses are required.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5379676)